tracing/tracers/basic/basic.go

```diff
@@ -4,13 +4,25 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"sync"
 	"time"
 
 	basic "github.com/opentracing/basictracer-go"
 	opentracing "github.com/opentracing/opentracing-go"
 )
 
-func InitTracer(opts []string) (opentracing.Tracer, error) {
+type CloseableTracer interface {
+	opentracing.Tracer
+	Close()
+}
+
+type BasicTracer struct {
+	tracer opentracing.Tracer
+	quit   chan struct{}
+	once   sync.Once
+}
+
+func InitTracer(opts []string) (CloseableTracer, error) {
```

Comment:
This would be API breaking change

Reply:
This is only basic tracer which is used only for tests. There is no value having this in any production workload, so I think it's fine to break API here.

Reply:
Ok, I am fine to have it like that